### ISSUE TITLE
Bootstrap 2

### ISSUE THREAD

**Issue by matsimitsu** [anjlab/bootstrap-rails#15]
Are there any plans to use bootstrap 2 for this gem?

**Comment by yury** [comment]
Working on it. Will release today

**Comment by matsimitsu** [comment]
Awesome, thanks!

**Comment by yury** [comment]
Just released anjlab-bootstrap-rails v 2.0.0.1, please test it.

`gem 'anjlab-bootstrap-rails', require: 'bootstrap-rails'`

**Comment by matsimitsu** [comment]
I just tested it and it raises an error on the following

```
Expected a color. Got: transparent
```

Trace:

```
anjlab-bootstrap-rails (2.0.0.1) vendor/assets/stylesheets/mixins.scss:426:in `gradient-striped'
anjlab-bootstrap-rails (2.0.0.1) vendor/assets/stylesheets/progress-bars.scss:57
anjlab-bootstrap-rails (2.0.0.1) vendor/assets/stylesheets/bootstrap.scss:56
```

Line 426 contains the following:

```
  background-image: linear-gradient($angle, rgba(255,255,255,.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,.15) 50%, rgba(255,255,255,.15) 75%, transparent 75%, transparent);
```

If i remove the line everything works.

**Comment by yury** [comment]
What version of SASS Gem do you use? 
I don't have this error with 

```
sass (3.1.13)
sass-rails (3.2.4)
```

**Comment by matsimitsu** [comment]
It seems the versions are the same

```
  * anjlab-bootstrap-rails (2.0.0.1)
  * sass (3.1.13)
  * sass-rails (3.2.4)
```

**Comment by yury** [comment]
Hmm, can you gist your Gemfile and Gemfile.lock files?
I want to reproduce it on my side.

Thanks

**Comment by matsimitsu** [comment]
No problem: https://gist.github.com/1737791

**Comment by yury** [comment]
Tried yours Gemfile and Gemfile.lock and everything works....

What version of ruby do you use?
I'm testing on ruby 1.9.3-p0, OSX Lion

PS: You don't need capistrano-ext gem any more. It is merged into capistrano gem.

**Comment by matsimitsu** [comment]
I've tried it in two projects. one is ruby ruby-1.8.7-p352 and one is ruby-1.9.3-preview1
both on OSX Snow Leopard

**Comment by yury** [comment]
Oh, just rm -rf ~/tmp/cache and got your error.

compass gem is causing this error.
Thinking for workaround.

**Comment by yury** [comment]
Replaced transparent with rgba(0,0,0,0) https://github.com/chriseppstein/compass/issues/356

Released 2.0.0.2

Note: rm -rf tmp/cache after upgrade
Also if you don't have sprites on buttons (http://twitter.github.com/bootstrap/base-css.html#icons) use gem 'compass', '>= 0.12.rc.0'

**Comment by matsimitsu** [comment]
Works like a charm, thanks!
